docs-markdown/viz.md

```diff
@@ -0,0 +1,284 @@
+## class CircleViz
+
+The `CircleViz` class handles the creation of a circle map and is built on top of the `MapViz` class.
+
+### Params
+**CircleViz**(_data, label_property=None, color_property=None, color_stops=None, color_default='grey', color_function_type='interpolate', *args, **kwargs_)
+
+Parameter | Description
+--|--
+data | name of GeoJson file or object
+label_property | property to use for marker label
+color_property | property to determine circle color
+color_stops | property to determine circle color
+color_default | property to determine default circle color if match lookup fails
+color_function_type | property to determine `type` used by Mapbox to assign color
```

Comment:
Add that values are one of `interpolate` or `match`.  The default is `interpolate`.